### PR TITLE
[TS-4369]Add VIA_ERROR_MOVED_TEMPORARILY

### PR DESCRIPTION
**Pull request by zizhong**
add VIA_ERROR_MOVED_TEMPORARILY to deal with redirect by remapping correctly for instrumentation (logging, metrics)

### CODE REVIEW

**Comment by JamesPeach** [comment]
Please update ``traffic_via`` and any other places that document or reference the Via string.

**Comment by zizhong** [comment]
Thanks @JamesPeach  for reminding me.
traffic_via and docs are updated.

File: doc/admin-guide/plugins/escalate.en.rst [hunk 1]
```diff
@@ -61,4 +61,4 @@ With this line in :file:`remap.config` ::
 Traffic Server would accept a request for ``cdn.example.com`` and, on a cache miss, proxy the
 request to ``origin.example.com``. If the response code from that server is a 401, 404, 410,
 or 502, then Traffic Server would proxy the request to ``second-origin.example.com``, using a
-Host: header of ``cdn.example.com``.
\ No newline at end of file
+Host: header of ``cdn.example.com``.
```
Comment (bgaff) [inline doc/admin-guide/plugins/escalate.en.rst]: Can you please `git checkout -- doc/admin-guide/plugins/esclate.en.rst` as I don't think you meant to change this file.

Comment (zizhong) [inline doc/admin-guide/plugins/escalate.en.rst]: Thanks Brian for picking this up.
Already revert that change.

**Comment by bgaff** [comment]
Merged.

**Pull request closed without merging**